### DISCUSSION TITLE
Update Problem from 2.5.1 to 2.6.1 - All data missing

### DISCUSSION THREAD

**Discussion by manny4566** [influxdata/influxdb#24020] [Q&A] [answered]
Hello,

i have a upgrade problem with influxdbv2 version 2.5.1 to 2.6.1.
After the upgrade over "apt-get- upgrade" all data, buckets and tokens are deleted or not accessible.

Before the upgrade everything works fine.
"influx org list" or "influx bucket list" works normal.

The login works:
![grafik](https://user-images.githubusercontent.com/23473485/210259804-c0a6a0ef-46cb-4685-b0c4-f4320790fe6c.png)

After the update to 2.6.1 everything is missed and the upgrade don't have keep the data.

Before:
```
root@srv:~# influxd version
InfluxDB v2.5.1 (git: 5b6fdbf05d) build_date: 2022-11-02T18:06:28Z
```

After:
```
root@srv:~# influxd version
InfluxDB v2.6.1 (git: 9dcf880fe0) build_date: 2022-12-29T15:53:07Z

```

Nothing works anymore:

```
root@srv:~# influx org list
Error: failed to list orgs: 401 Unauthorized: unauthorized access
```

The Web frontend shows the initial setup page:
![grafik](https://user-images.githubusercontent.com/23473485/210260086-a46b8b5f-e482-456c-901d-7ee5451251e6.png)

```
cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

What could be the problem ?
I hope you can help me. Otherwise i can not do a update.
Backup and restore don't work, because the restore can not fond the organization after the update.

Thank you

**Comment by jeffreyssmith2nd** [comment, marked as answer]
Can you check `/var/lib/influxdb` and make sure it has a directory structure that looks like https://docs.influxdata.com/influxdb/v2.6/reference/internals/file-system-layout/?

**Reply by manny4566** [reply]
> Can you check `/var/lib/influxdb` and make sure it has a directory structure that looks like https://docs.influxdata.com/influxdb/v2.6/reference/internals/file-system-layout/?

You help me with this link. Sometimes you can't see a simple error :D
The file "/etc/influxdb/config.toml" was not present on my system. And my data was stored under "/var/lib/influxdb/.influxdbv2".
After the upgrade, the config.toml file with the general path settings was created with "/var/lib/influxdb/" and not with "/var/lib/influxdb/.influxdbv2".
So influxdbv2 created an empty database. 
I migrated my data one directory higher to "/var/lib/influxdb" and now everything works.
I don't know why this is happening. Maybe an error during the migration from v1 to v2.
Now I am back to the default settings. 
Thank you very much.

**Reply by jeffreyssmith2nd** [reply]
Great, the simplest fixes are the best fixes!
